sympy/tensor/tensor.py

```diff
@@ -54,6 +55,24 @@ def deprecate_data():
     pass
 
 
+@deprecated(useinstead="TensorSymmetry constructor and methods", issue=17108,
+            deprecated_since_version="1.5")
+def deprecate_tensorsymmetry():
+    pass
+
+
+@deprecated(useinstead="TensorHead constructor or tensor_heads()", issue=17108,
+            deprecated_since_version="1.5")
+def deprecate_tensorhead():
```

Comment:
why not decorating `tensorhead`?

Reply:
I thought having deprecation declarations on top is common practice, my bad. I decorated ``tensorhead`` as you suggested.